src/views/account-details/account-details.view.tsx

```diff
@@ -227,11 +316,13 @@ function AccountDetails({
             }
```

Comment:
Can't we assume that we also want to show the Spinner when these async tasks are `pending`?

If so, we could do:

```typescript
if (
  !isAsyncTaskDataAvailable(accountTask) ||
  !isAsyncTaskDataAvailable(poolTransactionsTask) ||
  !isAsyncTaskDataAvailable(historyTransactionsTask) ||
  !isAsyncTaskDataAvailable(exitsTask)
) {
  return <Spinner />;
}

if (
  wallet &&
  isAsyncTaskDataAvailable(coordinatorStateTask) &&
  isAsyncTaskDataAvailable(fiatExchangeRatesTask)
) {...}
```

Reply:
I thought about that but maybe pending is not a case that we want to show the spinner because is not going to change to loading until other thing happens, I'm not sure about it

Reply:
We agreed to proceed with this change for 2 reasons:

* All the async tasks are launched at page load
* The `pending` status was rendering a fragment (now it renders the spinner)